opendbc/safety/safety/safety_toyota.h

```diff
@@ -15,6 +15,8 @@
   TOYOTA_BASE_TX_MSGS \
   {0x2E4, 0, 8, true}, {0x131, 0, 8, false}, \
   {0x343, 0, 8, false},  /* ACC cancel cmd */  \
+  {0x183, 0, 8, true},  /* ACC_CONTROL_2 */  \
+  {0x750, 0, 8, false},  /* radar diagnostic address */  \
```

Comment:
why is this needed? are we knocking out the radar?

Reply:
@adeebshihadeh  Here's is the context to why that is there: https://discord.com/channels/469524606043160576/1332212334147670048/1332214110787600434

We don't need to disable radar but the tests assume 0x750 is left open. This didn't seem specific to secoc so I didn't touch it and just added 750. Would you like anything changed in that test?

Reply:
@adeebshihadeh Fixed